system/services/src/com/percussion/services/audit/impl/PSDesignObjectAuditService.java

```diff
@@ -99,10 +103,12 @@ public void deleteAuditLogEntriesByDate(Date beforeDate)
       
       Session session = getSession();
 
-         Criteria criteria = session.createCriteria(PSAuditLogEntry.class);
-         criteria.add(Restrictions.lt("auditDate", beforeDate));
-         List<PSAuditLogEntry> entries = criteria.list();
-         
+      CriteriaBuilder builder = getSession().getCriteriaBuilder();
+      CriteriaQuery<PSAuditLogEntry> criteria = builder.createQuery(PSAuditLogEntry.class);
+      Root<PSAuditLogEntry> critRoot = criteria.from(PSAuditLogEntry.class);
+      criteria.where(builder.equal(critRoot.get("auditDate"),beforeDate));
```

Comment:
The new restriction in the criteria is using "equal" but the old one was using "lt" or less than.  This needs fixed so that the new one matches the old one otherwise the behavior of this method will have changed.